install_config/configuring_authentication.adoc

```diff
@@ -705,11 +715,17 @@ Basic Authentication is a generic backend integration mechanism that allows
 users to log in to {product-title} with credentials validated against a remote
 identity provider.
 
+Because basic authentication is generic, you can use this identity
+provider for advanced authentication configurations, such as:
+* xref:../install_config/sssd_for_ldap_failover.adoc.adoc#setting-up-for-ldap-failover[LDAP failover]
+* link:https://github.com/openshift/request-header-saml-service-provider[SAML authentication]
```

Comment:
Moving to line 948

Reply:
Flow does not seem correct.  I would think it would be near `provide Kerberos, LDAP, and many other forms of enterprise authentication` or at the end (so right before `[[GitHub]]`).

Reply:
@enj, it doesn't really work anywhere, but if you think it's better up a paragraph, done. :)